app/workers/post_process_media_worker.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class PostProcessMediaWorker
+  include Sidekiq::Worker
+
+  def perform(media_attachment_id)
+    media_attachment = MediaAttachment.find(media_attachment_id)
+    media_attachment.processing = :in_progress
+    media_attachment.save
+    media_attachment.file.reprocess!
```

Comment:
Does that mean the thumbnail will be re-processed?

Reply:
Yeah, I think I could limit it but maybe it's not a big deal. I'll look into it